python/eos/analysis.py

```diff
@@ -606,7 +606,7 @@ def sample_nested(self, bound='multi', nlive=250, dlogz=0.05):
         :type bound: str, optional
         :param nlive: The number of live points.
         :type nlive: int, optional
-        :param dlogz: Relative tolerance for the remaining evidence. Defaults to 5%.
+        :param dlogz: The relative tolerance for the remaining evidence. Defaults to 5%.
```

Comment:
Please move this from this commit to the previous commit, which introduces the method.